SU-SWS/ood-giving-campaign
# GIVCAMP-295 | Add Icon type and animation options to CTA

# READY FOR REVIEW


# Summary
- Add more icon types and new option icon animation for the CTA component in Storyblok

# Review By (Date)
- Retro

# Criticality
- 3
# Review Tasks

## Setup tasks and/or behavior to test

1. Pull down locally
2. Go to test/yvonne-test-volunteer-leadership-summit
3. Scroll to bottom of the photo gallery
4. Check that you can now pick more icons, also pick animation for the icon for the CTA component

![Screenshot 2024-03-04 at 4 24 11 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/09e70c44-fc97-43e8-bcfb-2f64c2885442)
![Screenshot 2024-03-04 at 4 24 36 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/1c1325ce-92b1-4f5c-a35a-0764250935cc)

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-295
